minerl/Malmo/Schemas/Mission.xsd

```diff
@@ -173,6 +173,18 @@
                         </xs:choice>
                     </xs:complexType>
                 </xs:element>
+                <xs:element name="StartingHealth" minOccurs="0" maxOccurs="1">
+                    <xs:complexType>
+                        <xs:attribute name="health" type="xs:float" use="optional"/>
+                        <xs:attribute name="maxHealth" type="xs:float" default="20" />
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="StartingFood" minOccurs="0" maxOccurs="1">
+                    <xs:complexType>
+                        <xs:attribute name="food" type="xs:int" default="20" />
+                        <xs:attribute name="foodSaturation" type="xs:float" use="optional" />
+                    </xs:complexType>
+                </xs:element>
```

Comment:
Is it possible to to make all these values optional, just for flexibility?

Reply:
Actually they are both optional: the first one because it has a default value (so you don't need to specify it) and the second one because it's marked as "optional". Basically the difference between "optional" and "default" is that "optional" seems to be the way to make it so that the default value is null.

Reply:
Yeah, I realized that later in my review...